library/ccm.c

```diff
@@ -358,7 +358,8 @@ int mbedtls_ccm_auth_decrypt( mbedtls_ccm_context *ctx, size_t length,
  */
 
 #define NB_TESTS 3
-
+#define PLAINTEXT_MAX_LENGTH  24
```

Comment:
Can we make these symbols a bit more specific to the thing they're doing and used for? eg. `CCM_SELFTEST_PT_MAX_LEN`, `CCM_SELFTEST_CT_MAX_LEN`.